tests/unit/Futurelink.spec.js

```diff
@@ -0,0 +1,77 @@
+import {shallowMount} from '@vue/test-utils'
+import Futurelink from '@/Futurelink';
+import {
+  createComponent,
+  createLink,
+  createRoute,
+  createWrapper
+} from '../helpers';
+
+describe('Futurelink.vue', () => {
+
+  test('throws when not using vue-router', () => {
+    expect(() => {
+      const error = console.error;
+      console.error = () => {
+      };
+      try {
+        shallowMount(Futurelink)
+      }
+      catch (e) {
+        console.error = error;
```

Comment:
It's a very minor thing, but I think this line should be in a `finally` block instead. At the moment, if an error isn't thrown, `console.error` will remain overridden.